sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -23,7 +23,7 @@ import scala.collection.immutable
 import scala.reflect.runtime.universe.TypeTag
 
 import org.apache.spark.{SparkConf, SparkContext}
-import org.apache.spark.annotation._
+import org.apache.spark.annotation.{DeveloperApi, Experimental, Stable, Unstable}
```

Comment:
So, is the only remaining instance `def experimental` which is natural to be `Experimental`?
```scala
  /**
   * :: Experimental ::
   * A collection of methods that are considered experimental, but can be used to hook into
   * the query planner for advanced functionality.
   *
   * @group basic
   * @since 1.3.0
   */
  @Experimental
  @transient
  @Unstable
  def experimental: ExperimentalMethods = sparkSession.experimental
```

Reply:
Pretty much. This one looks inherently experimental, so, I left it.